tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_report.py

```diff
@@ -0,0 +1,60 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+from marionette_driver import expected, By, Wait
+from gaiatest import GaiaTestCase
+from gaiatest.apps.base import Base
+from gaiatest.apps.messages.app import Messages
+
+
+class TestSmsReport(GaiaTestCase):
+
+    def test_sms_report(self):
+        
+        _text_type = (By.CSS_SELECTOR, 'p[data-l10n-id="message-type-sms"]')
+        _sender = (By.CSS_SELECTOR, 'a[data-name]')
+        _receiver = (By.CSS_SELECTOR, 'a.suggestion.js-contact-info')
+        
+        time_sent = time.time()
```

Comment:
I'd move that initialization right before `tap_send()`. Imagine the message app takes 2 minutes to launch => The test will fail, but not for the right reason. We'll report that the SMS report doesn't reflect the time where it has been sent.